lldb/source/Plugins/ExpressionParser/Swift/SwiftREPL.cpp

```diff
@@ -262,9 +263,8 @@ lldb::REPLSP SwiftREPL::CreateInstanceFromDebugger(Status &err,
   // Disable the cleanup, since we have a valid repl session now.
   cleanup.release();
 
-  std::string swift_full_version(swift::version::getSwiftFullVersion());
-  printf("Welcome to %s.\nType :help for assistance.\n",
-         swift_full_version.c_str());
+  printf("You are now in an interactive environment powered by the LLDB debugger where you can type, run, and debug Swift code.\n\n");
```

Comment:
Not sure about this message, seems a bit superfluous in my opinion.

Reply:
I went over this a little bit in the forum post. The reason for this message is that a lot of folks "end up" in the REPL, either intentionally or unintentionally through some default behavior. For folks learning Swift for the first time or someone who doesn't have a lot of command line experience with Swift, it can be rather mysterious as to what they are looking at. I thought it was also nice to call out that this is actually LLDB and not some other thing, so people start to understand the context they are in.

Again, if the REPL becomes an explicit choice, then of course the user should know where they are.